MiniApp/Classes/MiniApp.swift

```diff
@@ -47,7 +47,9 @@ public class MiniApp: NSObject {
     ///                         to interact with View component of mini app.
     ///         -   Error: Error details if Mini App View creating is failed
     ///   - messageInterface: Protocol implemented by the user that helps to communicate between Mini App and native application
-    public func create(appId: String, completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void, messageInterface: MiniAppMessageProtocol) {
+    public func create(appId: String,
+                       completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void,
+                       messageInterface: MiniAppMessageProtocol) {
```

Comment:
this line is a good example that we need to unify our class naming. For swift developers it's more familiar to deal with delegates when using interfaces or protocols. I know it's 3 different names for the same thing, but this is making use of the delegate pattern, so would it make more sense to use something like `messageDelegate: MiniAppMessageDelegate`. What do you thing about it @corycaywood and @leorjoseph ?

Reply:
I completely agree with this. We should definitely unify them, but I wonder that this could break the existing host app which has integrated them already. We should handle it a little carefully. but haven’t you done this change in the other PR of yours??

BTW, you are tagging the wrong person here, please use **@ts-leojoseph-r**  😆